.github/workflows/playwright.yml

```diff
@@ -0,0 +1,32 @@
+name: Playwright Tests
+on:
+  push:
+    branches: [ main, master ]
+  pull_request:
+    branches: [ main, master ]
+jobs:
+  test:
+    timeout-minutes: 60
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v4
+    - uses: actions/setup-node@v3
+      with:
+        node-version: 18
+    - name: Install dependencies
+      run: npm ci
+    - name: Build SDK
+      run: npm run build
+    - name: Install example dependencies
+      run: npm ci
+      working-directory: examples/minimal-react
+    - name: Install Playwright Browsers
+      run: npx playwright install --with-deps
+    - name: Run Playwright tests
+      run: npx playwright test
```

Comment:
I think we could invoke the script from `package.json`, for example, using `npm run e2e` instead of `npx playwright test`